src/main/resources/db/migration/V3_1__add_expired_varsel_published_at_and_svarfrist_to_aktivitetskrav_varsel.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE AKTIVITETSKRAV_VARSEL
+    ADD COLUMN expired_varsel_published_at TIMESTAMPTZ DEFAULT NULL;
```

Comment:
Trenger man ha med `varsel` i navnet her? Siden kontekst er varsel-tabellen :thinking:

Reply:
For tydelighetens skyld syns jeg det er greit, også leser det mye bedre. Kunne vært noe annet enn selve varselet som hadde utgått. Om jeg hadde kikket i en tabell der det hadde stått `expired_published_at` kunne jeg fint lurt på hva som egentlig var expired

Reply:
Enig i at det leser bedre, så kjør på :+1: